manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
The shebang line seems to have an incorrect path. It should be `#!/usr/bin/env python3` to ensure that Python 3 is used.